### PR TITLE
Add Vertical Navigation

### PR DESCRIPTION
**Pull request by fspoettel**
#139

### CODE REVIEW

File: docs/src/app/pages/VerticalNavigation/ExampleOverflow.js [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+          <VerticalNavigationItem href="#" isActive>Recent</VerticalNavigationItem>
+          <VerticalNavigationItem href="#">Created By Me</VerticalNavigationItem>
+          <VerticalNavigationItem href="#">Private Reports</VerticalNavigationItem>
+          <VerticalNavigationItem href="#">All Reports</VerticalNavigationItem>
+        </VerticalNavigationSection>
+        <VerticalNavigationOverflow
+          id="search-results"
+          isOpen={overFlowOpen}
+          label="Reports"
+          onToggle={this.onOverflowToggle}
```
Comment (PeterSiegl) [inline docs/src/app/pages/VerticalNavigation/ExampleOverflow.js]: `title` prop is required

File: src/components/Form/Input.js [hunk 2]
```diff
@@ -69,7 +69,7 @@ Input.defaultProps = {
   role: null,
   required: false,
   type: 'text',
-  value: null,
+  value: undefined,
```
Comment (PeterSiegl) [inline src/components/Form/Input.js]: 👍

File: docs/src/app/pages/VerticalNavigation/ExampleDefault.js [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+ } from 'react-lds';
+
+const searchInput = (
+  <Input
+    hideLabel
+    id="search"
+    label="search"
+    placeholder="Quick Search"
+    type="text"
+    value={undefined}
```
Comment (PeterSiegl) [inline docs/src/app/pages/VerticalNavigation/ExampleDefault.js]: you could add `iconLeft="search"`

**Comment by PeterSiegl** [comment]
When testing I got the following warning, most likely some global issue: `Warning: Failed prop type: The prop onClick is marked as required in Flavored_Button, but its value is undefined.  in Flavored_Button`

**Comment by fspoettel** [comment]
@PeterSiegl yes, we need to track down that warning

**Pull request merged**